numba_cuda/numba/cuda/cudadrv/driver.py

```diff
@@ -2930,6 +3003,113 @@ def complete(self):
         return bytes(np.ctypeslib.as_array(cubin_ptr, shape=(size,)))
 
 
+class PyNvJitLinker(Linker):
+    def __init__(
+        self,
+        max_registers=None,
+        lineinfo=False,
+        cc=None,
+        lto=False,
+        additional_flags=None,
+    ):
+        if pynvjitlink_import_err is not None:
+            raise ImportError(_MVC_ERROR_MESSAGE_CU12)
```

Comment:
I think this could be removed / moved to the check for whether pynvjitlink should be enabled above, so we report a setup / config error to the user right away.